sw/device/silicon_creator/rom_ext/imm_section/utils.bzl

```diff
@@ -95,6 +102,20 @@ create_imm_section_targets(
     name = "{name}",
     src = "{filename}",
 )
+
+otp_json_immutable_rom_ext(
```

Comment:
Why are you generating this?

This tooling should be very simple. Build a imm_section bin and thats it. The bin then gets checked in to a repo. OTP settings have nothing to do with this release flow. SKU owners will configure OTP settings out of band.